packages/scss/src/components/_button.scss

```diff
@@ -336,47 +335,45 @@
 	}
 
 	/* Loading */
-
 	&.is-loading {
-		@include loading(_component("button.font-size"));
-		color: transparent;
-		opacity: _theme("commons.disabled.opacity");
+		@include loading(_component('button.font-size'));
+		color: transparent !important;
```

Comment:
Could we use specificity in a way that would avoid using !important here?

Reply:
When there was no `!important` it caused a bug when combining a palette and a state (two classes).

![Capture d’écran 2022-02-23 à 11 27 48](https://user-images.githubusercontent.com/64789527/155301615-3988df65-5fab-47cb-9a90-fc492924212f.png)